bin/export.py

```diff
@@ -243,7 +243,10 @@ def export_contest():
     if not has_ryaml:
         for key in ('duration', 'scoreboard_freeze_duration'):
             if key in data:
-                data[key] = str(datetime.timedelta(seconds=data[key]))
+                value = data[key]
+                if isinstance(value, int):
+                    str(datetime.timedelta(seconds=data[key]))
+                data[key] = value;
```

Comment:
hmm are strings supported here?

Reply:
pyyaml seems to pars "0:xx:xx" as string but "x:xx:xx" as int

Reply:
huh what?? it parses an `hh:mm:ss` duration as a time in seconds??